Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb

```diff
@@ -4,7 +4,10 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    Dont rush! take time to read the text and then run the next five code cells only one time each
</pre><p>Remove moving text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ignaciomigliaro/Group_theory_for_IR/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>